Wbooks/Detail/Comments Section/CommentCell.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CommentCell.swift
+//  Wbooks
+//
+//  Created by Juan Martín Gordo on 26/04/2021.
+//
+
+import UIKit
+
+class CommentCell: UITableViewCell {
+    
+    // MARK: Properties
+    static let identifier = "CommentCell"
+    private var request: DispatchWorkItem?
+    
+    @IBOutlet weak var cellView: UIView!
+    @IBOutlet weak var userImage: UIImageView! {
+        didSet {
+            userImage.layer.cornerRadius = userImage.frame.height/2
+            userImage.clipsToBounds = true
+        }
+    }
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var commentLabel: UILabel!
+    
+    // MARK: Actions
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        self.backgroundColor = .clear
+    }
+    
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        request?.cancel()
+        request = nil
+        userImage.image = UIImage(named: "User1")
```

Comment:
As we once did with `UIColor`, we could create a `UIImage` extension with the images we are using throughout the app.